common/lib/common-utils/src/assert.ts

```diff
@@ -12,6 +12,8 @@
  * A number should not be specified manually: use a string.
  * Before a release, policy-check should be run, which will convert any asserts still using strings to
  * use numbered error codes instead.
+ *
+ * @deprecated - Use the exported function the core-utils package.
```

Comment:
Nit: `@deprecated` comments don't want the trailing `-`.
```suggestion
 * @deprecated Use the exported function the core-utils package.
```

Reply:
ARRRRRRGGGGGHHH!!! :smile:

Reply:
I'll fix those in this PR, but I also opened #16935 for other offenses across the repo.

Reply:
This was the only one I saw FWIW